### ISSUE TITLE
Change default theme to grey

### ISSUE THREAD

**Issue by elstoc** [darktable-org/darktable#11435]
We all know dark themes are bad for editing photos and we recommend against them in the user manual but, for some reason, the default theme is still the "darktable" theme. Since we recommend grey themes, we should use one of them as the default (darktable-elegant-grey?)

I hesitate to suggest it but we should probably update all our screenshots as well :(

**Comment by TurboGit** [comment]
No strong opinion! We had chosen the dark as medium as a consensus. I personally find the gray a bit too light to my taste. But that's only my taste, so whatever got the most votes will be ok by me as long as I can choose the version I prefer.

**Comment by difrkaguilar** [comment]
I know that gray theme is the recommended but as @TurboGit said as long as I can use my personal "dark" theme I'll.
Two days ago I buy a new BenQ monitor 32" and for me, the gray theme is too light.

**Comment by elstoc** [comment]
Each to their own but it's not about what looks good but rather what makes your editing better, and a mid-grey background avoids certain adverse impacts on visual perception. See [the user manual](https://docs.darktable.org/usermanual/3.8/en/module-reference/utility-modules/darkroom/color-assessment/) for details.

**Comment by AlicVB** [comment]
I'm a grey theme only user. But same for the others, I don't care if I need to change it in the prefs...
As I see this : 
pros: 
- better for editing
- inline with dt philosophy : propose the best things we can, even if they are not the more appealing

cons:
- more different from other app (tbh, I see that as a pro too)
- we will certainly get more complains :)

guess what ? I vote for grey theme as default :)

**Comment by ptilopteri** [comment]
I also use grey but spend little time in the side panels.  When working
a set, I usually hide the left panel and use only the right to activate modules.

**Comment by Nilvus** [comment]
We are at least all agree that is dark or grey theme as default and not darker one. Which is good. I use grey theme (of course, I've done it!). I agree with @elstoc that grey theme should be the default as it could allow more users to have the good theme for better editing (we talk here about better editing and visual perception of course, not which is one the best for contrast of items in the UI ; this will never be grey theme by definition).

And  I understand so why dark theme had been chosen and if we keep it.

Said that, I would go for grey theme as having more place for the image is what we tried since darktable 3.0 for the UI. And this goes in that direction by allowing better theme for editing (grey one so).

But to help that and we confirm that direction, I would suggest 2 things:

- of course, don't change theme already set in darktable by the user
- so only set the new default theme for new install AND on first launch of darktable have a dialog window with a little text explain that choice and a link to the user manual page WITH a menu to change theme.

**Comment by elstoc** [comment]
> AND on first launch of darktable have a dialog window with a little text explain that choice and a link to the user manual page WITH a menu to change theme.

Not sure about this.

**Comment by Nilvus** [comment]
Was just an idea.

**Comment by difrkaguilar** [comment]
Well, if the idea is to follow the rules by the book, then grey should be the default theme once the program starts, then if other users wish to switch to other themes it is by their choice.

> * of course, don't change theme already set in darktable by the user

Without a doubt, the possibility for users to customize previously created themes or modify existing ones should be continued.

> * so only set the new default theme for new install AND on first launch of darktable have a dialog window with a little text explain that choice and a link to the user manual page WITH a menu to change theme.

This could be an idea, I've been thinking for a while that darktable could include a window at startup as other programs like Blender and Inkscape do (that could also come up only once and then the changes that are not made in it can be made in the darktable configuration window) where the predefined theme is shown, as well as some basic aspects of the configuration. This would be in accordance with the post #11420 where a basic and an advanced (expert) configuration is desired.

**Comment by Nilvus** [comment]
When using middle grey, contrast is automatically worse than black one, but it's also much better to use middle grey for correct processing.

What is good is that changing theme is quick and easy.

**Comment by kmilos** [comment]
I'm just worried there will be quite a few new users who will open dt for the first time, go "yuck, can't read a thing here", don't know/bother about going into the settings and never use dt again.

A much better approach would've been to force some first/splash screen and to force the user select their theme on first start rather than defaulting to something not ideal (I understand it has advantages for processing though).

**Comment by elstoc** [comment]
We should give those users their money back. Seriously though, this is an image processing application and we should practice what we preach when we recommend the grey theme as the best to use. I would not want to pick this particular setting as one to flag to the user "we've set this as default but we're not sure you'll like it so feel free to choose another". If users can't be bothered to work out how to change the theme then I'm not sure we should be hand-holding them to choose a non-recommended setting.

**Comment by elstoc** [comment]
> Edit: And looks like I'm not the only one to notice this on Windows: https://discuss.pixls.us/t/darktable-windows-insider-program-5-29/30999/3

Seems to be more a comment about the font, than the colour scheme. Perhaps we should have another grey variant with the same font as the darktable theme.

**Comment by apostel338** [comment]
Yes, my comment is all about the font. At least my Windows seems to have a serious problem with the rendition of this “Segoe UI light”.

**Comment by kmilos** [comment]
> Seems to be more a comment about the font, than the colour scheme.

Same difference - I have no problem w/ gray, I have a problem w/ readability, as originally stated. There might be room to tweak the default theme.

**Comment by elstoc** [comment]
Sure but this is a separate issue (with the darktable-elegant-grey theme, which I didn't change) and should be raised separately.

**Comment by Nilvus** [comment]
I agree, font related issue is not about grey theme. Font for elegant themes are set inside darker elegant theme. dark and grey theme inherit from it. Note too that if Segoe font is used, that should means that you don't have Roboto font installed (could be installed easily from Google fonts repository. Roboto fonts are recommanded for better rendering of all elegant themes.

So if the issue is just font,on Windows, maybe something could be updated. @apostel338 or @kmilos: it would be good if you could post an issue about that (write Windows specific one please in it) and ping me when the issue is done.
I don't use Windows but with your feedback, we could maybe update darker CSS file like you suggest on pixls.us forum @apostel338.
I suggest to stop here about that font issue and continue when an issue will be posted.

**Comment by kmilos** [comment]
Agreed - @apostel338 would yo mind creating a new issue/PR as you already have all the screenshots available? Thanks

**Comment by apostel338** [comment]
Here's the new font related one: https://github.com/darktable-org/darktable/issues/11945
